app/uk/gov/hmrc/apinotificationqueue/repository/Message.scala

```diff
@@ -19,4 +19,5 @@ package uk.gov.hmrc.apinotificationqueue.repository
 import java.sql.Date
 import java.util.UUID
 
+// TODO: maybe move this to models.scala
```

Comment:
Can move this and ClientMessages to models in my opinion and just remove the comment.

Reply:
what I have done here is follow the layering used in https://github.com/hmrc/api-subscription-fields which I think worked well